packages/calypso-e2e/src/lib/pages/marketing-page.ts

```diff
@@ -11,13 +11,14 @@ import { Page } from 'playwright';
 
 const selectors = {
 	content: '#primary',
-	navtabsList: '.section-nav-tabs__list',
+	navTabs: 'div.section-nav-tabs',
+	navTabsDropdownOptions: '.select-dropdown__option',
```

Comment:
nit:
```suggestion
	navTabsDropdownOption: '.select-dropdown__option',
```

Reply:
I think I used a similar convention elsewhere in the codebase, where I used plural ending for selectors that will match against multiple items (eg. gallery items, select dropdown options, etc).

Would you say it is worth codifying to use singular naming where possible, and perhaps go back and remove these plural endings (lower priority item)?

Reply:
Yeah, let's keep the singular naming. I think plurals are generally used for containers/wrappers (`.nav-tabs`, `.comments`).